docs/csharp/index.yml

```diff
@@ -101,17 +157,13 @@ landingContent:
           - text: Version compatibility
             url: whats-new/version-update-considerations.md
 
-  - title: "C# language reference"
+  - title: "Stay in Touch"
     linkLists:
       - linkListType: reference
         links:
-          - text: "Language reference"
-            url: language-reference/index.md
-          - text: "C# keywords"
-            url: language-reference/keywords/index.md
-          - text: "C# operators"
-            url: language-reference/operators/index.md
-          - text: Configure language version
```

Comment:
These two could be removed to make this list smaller. They are easily findable from the language reference page.